src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs

```diff
@@ -342,6 +343,9 @@ private static void AddPopParams(AuthenticationRequestParameters authenticationR
                         authorityUrl = authorityUrl.Substring(0, authorityUrl.Length - "v2.0".Length);
                 }
 
+
```

Comment:
but I guess it is ok since this is in [MsalException.cs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/4307/files/af51387737fff6df2f083c2efc0a66faa4e21c35#diff-0149c194ca9a8c938bc5b4f234f0be30e5f6210f7a622477d753cbee66776fb6)